src/main/java/seedu/duke/notebooks/Section.java

```diff
@@ -50,16 +51,11 @@ public void getPage(int pageIndex) {
         }
     }
 
-    public Page removePage(int indexToRemove) {
-        return pageArrayList.remove(indexToRemove);
-    }
-
-    public void removePage(String titleToRemove) {
-        int indexToRemove = getPage(titleToRemove);
-        if (indexToRemove >= 0) {
-            removePage(indexToRemove);
-        } else {
-            System.out.println("doesn't exist");
```

Comment:
So it no longer removes pages based off titles as an input?

Reply:
it does remove based on `title`. it's just that the index of the page `entitled title` is passed into the removePage function